pyproject.toml

```diff
@@ -0,0 +1,142 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=64.0.0",
+    "tomli  ; python_version < '3.11'",
+    "versioneer>=0.24",
+]
+
+[project]
+name = "dask-cuda"
+dynamic = [
+    "version",
+]
+description = "Utilities for Dask and CUDA interactions"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license= { text = "Apache-2.0" }
+requires-python = ">=3.8"
+dependencies = [
+    "dask >=2022.9.2",
+    "distributed >=2022.9.2",
+    "pynvml >=11.0.0",
+    "numpy >=1.18.0",
+    "numba >=0.54",
+    "pandas >=1.0",
+    "zict >=0.1.3",
+]
+classifiers=[
+    "Intended Audience :: Developers",
+    "Topic :: Database",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+
+[project.scripts]
+dask-cuda-worker = "dask_cuda.cli.dask_cuda_worker:go"
+dask-config = "dask_cuda.cli.dask_config:go"
+
+[project.optional-dependencies]
+docs = [
+    "numpydoc",
+    "sphinx",
+    "sphinx-click",
+    "sphinx-rtd-theme",
+]
+test = [
+    "pytest",
+]
```

Comment:
Added an optional test requirement for clarity. Only saw `pytest` being used.

Not sure if `pytest-asyncio` is still needed ( or desired https://github.com/rapidsai/dask-cuda/issues/1044 )

Can add anything else needed here